PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -65,6 +65,11 @@ struct HfCandidateCreator2Prong {
   Configurable<bool> useSel8Trigger{"useSel8Trigger", true, "apply the sel8 event selection"};
   Configurable<float> zPvPosMax{"zPvPosMax", 10.f, "max. PV posZ (cm)"};
   Configurable<bool> useTimeFrameBorderCut{"useTimeFrameBorderCut", true, "apply TF border cut"};
+  Configurable<bool> useIsGoodZvtxFT0vsPV{"useIsGoodZvtxFT0vsPV", false, "check consistency between PVz from central barrel with that from FT0 timing"};
+  Configurable<bool> useNoSameBunchPileup{"useNoSameBunchPileup", false, "exclude collisions in bunches with more than 1 reco. PV"}; // POTENTIALLY BAD FOR BEAUTY ANALYSES
+  Configurable<bool> useNumTracksInTimeRange{"useNumTracksInTimeRange", false, "apply occupancy selection (num. ITS tracks with at least 5 clusters in +-100us from current collision)"};
+  Configurable<float> numTracksInTimeRangeMin{"numTracksInTimeRangeMin", 0, "min. value for occupancy selection"};
+  Configurable<float> numTracksInTimeRangeMax{"numTracksInTimeRangeMax", 1e+10, "max. value for occupancy selection"};
```

Comment:
```suggestion
  Configurable<int> numTracksInTimeRangeMin{"numTracksInTimeRangeMin", 0, "min. value for occupancy selection"};
  Configurable<int> numTracksInTimeRangeMax{"numTracksInTimeRangeMax", 1e+10, "max. value for occupancy selection"};
```
And elsewhere

Reply:
thanks @vkucera , done in the next commit